oem.json

```diff
@@ -3039,32 +3168,102 @@
+        ],
         "removal": "delete"
     },
     {
         "id": "com.motorola.android.fmradio",
-        "description": "FMRadioService \nRequired by \"com.motorola.fmplayer\"\n",
+        "label": "FMRadioService",
+        "dependencies": [
+            "com.motorola.fmplayer"
+        ],
+        "description": "Required for Motorola FM Radio",
         "removal": "delete"
     },
+    {
+        "id": "com.motorola.android.fota",
+        "label": "Software update",
+        "description": "Required for OTA updates which are ssential part of keeping your device secure and up to date with regular security patchs.\nFOTA = firmware over the air",
+        "removal": "caution",
+        "warning": "Breaks OTA updates"
+    },
     {
         "id": "com.motorola.android.jvtcmd",
-        "description": "Java Tcmd Helper\ntcmd = commandes types. Seems to be a tools wich help find Java commands types.\nUseless for normal user.\n",
+        "label": "Java Tcmd Helper",
+        "description": "Seems to be a tools wich help find Java commands types.\ntcmd = commandes types",
         "removal": "delete"
     },
     {
         "id": "com.motorola.android.nativedropboxagent",
-        "description": "Native DropBox Agent.\nIt is not related to Cloud Dropbox company but to Android logging. It is used during development.\nhttps://stackoverflow.com/questions/4434192/dropboxmanager-use-cases\n",
+        "label": "NativeDropBoxAgent",
+        "description": "It is not related to Cloud Dropbox company but to Android logging. It is used during development.",
+        "web": [
+            "https://stackoverflow.com/questions/4434192/dropboxmanager-use-cases",
+            "https://beta.pithus.org/report/b7376e9ca607e856c9b39eb93e5aa420dab7f32424b61c5325f55faa03d2a97f"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.android.overlay.payjoy",
+        "label": "com.motorola.android.overlay.payjoy",
+        "description": "Overlay for 'com.payjoy.access'",
         "removal": "delete"
     },
     {
         "id": "com.motorola.android.providers.chromehomepage",
-        "description": "Seems to provide the \"Home\"-button functionality in Chrome.\nhttps://forum.xda-developers.com/android/apps-games/app-chrome-homepage-t3695804",
+        "label": "com.motorola.android.providers.chromehomepage",
+        "description": "Seems to provide the \"Home\"-button functionality in Chrome.",
+        "web": [
+            "https://forum.xda-developers.com/android/apps-games/app-chrome-homepage-t3695804"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.android.providers.settings",
+        "label": "Settings storage",
+        "description": "Seems to required for device settings.",
+        "removal": "unsafe",
+        "warning": "Some of the device settings will crash continuously."
```

Comment:
Motorola has extra settings provider in addition to the AOSP. That's why it's required for ALL Motorola devices.